applications/serial_lte_modem/doc/TCPUDP_AT_commands.rst

```diff
@@ -57,13 +58,43 @@ Response syntax
 * The ``<error>`` value is an integer.
   It represents the error value according to the standard POSIX *errno*.
 
-Examples
+Unsolicited notification
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+   #XTCPSVR: "<peer ip>","connected"
+
+   This is emitted when a new connection has been created to the server.
+
+* The ``<peer_ip>`` value is the IPv4 or IPv6 address of the peer.
+
+
+Unsolicited notification
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tcpsvr_status_notif_start
+
+::
+
+   #XTCPSVR: <error>,"disconnected"
+
+   This is emitted when the client has been disconnected.
+
+* The ``<error>`` value is an integer of 0, for normal closure and *-errno* for erroneous closure.
```

Comment:
```suggestion
* The ``<error>`` value is an integer.
  It is either ``0`` when the server closed normally, or an *-errno* code.
```
Check that I got the meaning right.

Reply:
So this would be more correct, then:
```suggestion
* The ``<error>`` value is an integer.
  It is either ``0`` when the client disconnects normally, or an *-errno* code.
```

Reply:
The disconnected comes in both cases when the server closes the connection via hangup or when the client closes the connection.

So maybe like this:

  It is either ``0`` when the client is disconnected normally, or an *-errno* code.

Reply:
I'd say the first one is better, just to make sure that the timing is clear.